ansible/roles/superhero-dashboard/templates/configmap.yml.j2

```
@@ -7,6 +7,7 @@ metadata:
     app: shd
 data:
   BACKEND_URL: "http://api-svc:{{ PORT_SERVER }}/api/"
+  ADMIN_API_URL: "http://api-svc:4030/admin/api/"
```

change this line to api-admin-svc 
```suggestion
  ADMIN_API_URL: "http://api-admin-svc:4030/admin/api/"
```